python/ql/test/experimental/dataflow/coverage/dataflow.ql

```diff
@@ -1,5 +1,10 @@
-import experimental.dataflow.testConfig
+/**
+  * @kind path-problem
+  */
 
-from DataFlow::Node source, DataFlow::Node sink
-where exists(TestConfiguration cfg | cfg.hasFlow(source, sink))
-select source, sink
+ import experimental.dataflow.testConfig
```

Comment:
Slightly strange indent - autoformat?

Reply:
Hm, I would hope that it has been formatted, but it looks like not..

Reply:
Formatted now, at least..